### PR TITLE
Feature/store-past-deals

### PR DESCRIPTION
**Pull request by nmlinaric**
Created ```PUT``` endpoint ```/user/node/pastdeals``` for storing/updating past deals of a node. 
A deal has  one-to-one relation to a node.  Endpoint recieves: cid, state, size, provider, price, duration and url and address of the node to which it belongs.

### CODE REVIEW

File: src/Migrations/20200210114133-create-node-past-deals.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                type: sequelize.INTEGER
+            },
+            size: {
+                type: sequelize.STRING
+            },
+            provider: {
+                type: sequelize.STRING
+            },
+            price: {
+                type: sequelize.STRING
```
Comment (morrigan) [inline src/Migrations/20200210114133-create-node-past-deals.js]: shouldn't here be BIGINT?

File: src/Migrations/20200210114133-create-node-past-deals.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                type: sequelize.INTEGER
+            },
+            cid: {
+                type: sequelize.STRING
+            },
+            state: {
+                type: sequelize.INTEGER
+            },
+            size: {
+                type: sequelize.STRING
```
Comment (morrigan) [inline src/Migrations/20200210114133-create-node-past-deals.js]: BIGINT as well? cc @MakMuftic

File: src/Migrations/20200210114133-create-node-past-deals.js (hunk 1 above)
Comment (nmlinaric) [reply src/Migrations/20200210114133-create-node-past-deals.js]: Yes, my mistake. In the application it is a string, but in the database bigint.

File: src/Migrations/20200210114133-create-node-past-deals.js (hunk 2 above)
Comment (nmlinaric) [reply src/Migrations/20200210114133-create-node-past-deals.js]: Also, same thing as above.

**Pull request merged**